src/Packagist/WebBundle/Resources/views/user/enable_two_factor_auth.html.twig

```diff
@@ -0,0 +1,66 @@
+{% extends "PackagistWebBundle:user:packages.html.twig" %}
+
+{% import "PackagistWebBundle::macros.html.twig" as macros %}
+
+{% set isActualUser = app.user and app.user.username is same as(user.username) %}
+
+{% block content %}
+    <h2 class="title">
+        {{ user.username }}
+        <small>
```

Comment:
the `<small>` tag should be inside the `if` to avoid rendering an empty tag.

Reply:
This code was copied as-is from existing templates in that same folder.  I've fixed the instances in my templates and will submit a separate PR to address the others :)